fs-integration-test-suite/accelerator-tests/gateway-tests/payments/src/test/groovy/org/wso2/financial/services/accelerator/gateway/test/payments/Payment_Initiation_Tests/PaymentsInitiationRequestHeaderValidationTests.groovy

```diff
@@ -125,7 +125,7 @@ class PaymentsInitiationRequestHeaderValidationTests extends FSAPIMConnectorTest
     }
 
     //TODO: https://github.com/wso2/financial-services-accelerator/issues/681
```

Comment:
Let's remove the todo comments if the issue is fixed.

Reply:
This issue is not fixed hence kept as enabled false